src/components/ebay-confirm-dialog/examples/01-default/template.marko

```diff
@@ -0,0 +1,25 @@
+<ebay-confirm-dialog reject-text="Cancel" confirm-text="Okay" open=state.open on-reject("closeDialog") on-confirm('success')>
+    <@header>Heading</@header>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+    <p><a href="http://www.ebay.com">www.ebay.com</a></p>
```

Comment:
Nitpick, but please remove the www.ebay.com link. We are trying to restrict confirm and alert dialogs to just heading, description and button(s).